mods/beds/functions.lua

```diff
@@ -21,6 +21,14 @@ local function get_look_yaw(pos)
 	end
 end
 
+local function get_night_skip()
```

Comment:
I'm always open to suggestions about function names, althrought`get_night_skip` was shorter, `is_night_skip_enabled` sounds more evident. Thanks.
https://github.com/LeMagnesium/minetest_game/commit/c8049752275c5c609cbf4e02c8056fca863e5a94